scripts/feature-flag-audit.sh

```
@@ -0,0 +1,293 @@
+    'useFlag\s*\(\s*['"'"'"][^'"'"'"]+['"'"'"]\s*\)'
+    '<FeatureGate\s+flag=['"'"'"][^'"'"'"]+['"'"'"]'
+)
+
+PYTHON_PATTERNS=(
+    'is_enabled\s*\(\s*['"'"'"][^'"'"'"]+['"'"'"]\s*\)'
+    'feature_flags.*is_enabled'
+)
+
+# Temporary files
+TEMP_DIR=$(mktemp -d)
+trap "rm -rf $TEMP_DIR" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)